src/main/java/br/unicamp/meca/models/ActionSequencePlan.java

```diff
@@ -44,22 +44,36 @@ public ActionSequencePlan(ActionStep[] actionIdSequence) {
 	}
 	
 	/**
-	 * Returns the id of the current ActionFromPlanningCodelet to be undertaken in the plan.
+	 * Returns the ActionStep of the current ActionFromPlanningCodelet to be undertaken in the plan.
 	 * 
 	 * @return currentActionId
```

Comment:
This javadoc is wrong - return type has changed

Reply:
Changed in next commit